### ISSUE TITLE
Sharing buttons tooltip: doesn't close on tap

### ISSUE THREAD

**Issue by achinara** [dimsemenov/PhotoSwipe#1719]
Faced two problems:

1. Sharing buttons tooltip doesn't close when taping on background. Closed with a vertical swipe along with the gallery
2. If you tap on the "invisible share button", the tooltip will open

How to fix it?

![RPReplay-Final1600782379](https://user-images.githubusercontent.com/34182325/93893992-edccd580-fcf6-11ea-9108-8c082dc34a9f.gif)

**Comment by andi34** [comment]
https://codepen.io/andi34/pen/qBZRxgv

**Comment by achinara** [comment]
no, it didn't help, I tried with the same options as in your example

**Comment by andi34** [comment]
But does it work in the example? Or you've only adjusted the options in your code?

Could be you face the issue which is fixed by https://github.com/dimsemenov/PhotoSwipe/pull/1708 ?
